libraries/rush-sdk/src/index.ts

```diff
@@ -89,7 +92,30 @@ if (rushLibModule === undefined) {
   }
 }
 
-// SCENARIO 3:  A tool or script depends on "rush-sdk", and is meant to be used inside a monorepo folder.
+// SCENARIO 3: A tool or script has been invoked as a child process by an instance of "rush-lib" and can use the
+// version that invoked it. In this case, use process.env.RUSH_LIB_PATH to find "rush-lib".
+if (rushLibModule === undefined) {
+  const rushLibVariable: 'RUSH_LIB_PATH' = 'RUSH_LIB_PATH';
+  const rushLibPath: string | undefined = process.env[rushLibVariable];
+  if (rushLibPath) {
+    terminal.writeVerboseLine(
+      `Try to load ${RUSH_LIB_NAME} from process.env.${rushLibVariable} from caller package`
+    );
+    try {
+      rushLibModule = _require(rushLibPath);
+    } catch (error) {
+      terminal.writeVerboseLine(`Filed to load ${RUSH_LIB_NAME} via process.env.${rushLibVariable}`);
```

Comment:
Should this be a warning instead of a "verbose" message? It seems strange that `RUSH_LIB_NAME` could be specified but the target path is incorrect.  If you can think of a common reason why this can occur, then keep it as-is.